protoc-gen-openapiv2/options/openapiv2.proto

```diff
@@ -534,6 +534,8 @@ message JSONSchema {
   reserved 30 to 33;
   // Items in 'array' must be unique.
   repeated string array = 34;
+  // Only available in Open API v3, but some tools (like Redoc) support it for v2.
+  bool write_only = 99;
```

Comment:
I added it at number 99 since it's not in the openapi v2 spec, but can change that if needed

Reply:
I don't want to create this precedent, we don't support any openapiv3 properties in our openapiv2 generator. We support setting extensions, which have been used to backport some openapiv3 features. I think this is going too far.